src/include/duckdb/execution/ht_entry.hpp

```diff
@@ -21,10 +21,16 @@ namespace duckdb {
 */
 struct ht_entry_t { // NOLINT
 public:
+#if defined(__ANDROID__) || defined(ANDROID)
```

Comment:
```suggestion
#if defined(__ANDROID__)
```

This should be sufficient, the codebase doesn't use the `ANDROID` macro name.